compiler/src/dotty/tools/dotc/util/Collections.scala

```diff
@@ -0,0 +1,39 @@
+/* NSC -- new Scala compiler
```

Comment:
I doubt it. If we do want to keep these methods, we should also move them to Decorators.